internal/orchestrationapi/orchestrationapi.go

```diff
@@ -254,7 +254,7 @@ func getExecCmds(execType string, requestServiceInfos []RequestServiceInfo) ([]s
 		}
 	}
 
-	return nil, errors.New("Not Found")
+	return nil, errors.New("not found")
```

Comment:
Does all error messages should be written in lowercase?

Reply:
Staticcheck determines this as an [error: ST1005 – Incorrectly formatted error string](https://staticcheck.io/docs/checks#ST1005)